integrations/sensu/platform-discovery/sensu-resources.yaml

```diff
@@ -10,16 +10,16 @@ metadata:
 spec:
   command: sensu-platform-discovery
   runtime_assets:
-  - sensu/sensu-platform-discovery:0.0.2
+    - sensu/sensu-platform-discovery:0.0.2
   subscriptions:
-  - autodiscovery
+    - autodiscovery
   publish: true
   interval: 60
   timeout: 5
   pipelines:
-  - api_version: core/v2
-    type: Pipeline
-    name: entity-manager
+    - api_version: core/v2
```

Comment:
@calebhailey is the additional indentation intentional?

Reply:
Yes. We need to hook up a linter to enforce a style, but most of the linters expect YAML arrays to be indented, so I've been trying to fix these up as I go.